gammapy/modeling/fit.py

```diff
@@ -513,6 +559,24 @@ def stat_contour(self, datasets, x, y, numpoints=10, sigma=1):
         result : dict
             Dictionary containing the parameter values defining the contour, with the
             boolean flag "success" and the information objects from ``mncontour``.
+
+        Examples
+        --------
+        >>> from gammapy.datasets import Datasets, SpectrumDatasetOnOff
+        >>> from gammapy.modeling.models import SkyModel, LogParabolaSpectralModel
+        >>> from gammapy.modeling import Fit
+        >>> datasets = Datasets()
+        >>> for obs_id in [23523, 23526]:
+        ...     dataset = SpectrumDatasetOnOff.read(
+        ...         f"$GAMMAPY_DATA/joint-crab/spectra/hess/pha_obs{obs_id}.fits"
+        ...     )
+        ...     datasets.append(dataset)
+        >>> datasets = datasets.stack_reduce(name="HESS")
+        >>> model = SkyModel(spectral_model=LogParabolaSpectralModel(), name="crab")
+        >>> datasets.models = model
+        >>> fit = Fit(backend='minuit')
+        >>> optimize = fit.optimize(datasets)
+        >>> stat_contour = fit.stat_contour(datasets=datasets, x="alpha", y="amplitude")
```

Comment:
having `x` and `y` as `parameter` directly is probably clearer

Reply:
done